Makefile

```diff
@@ -132,7 +132,7 @@ manifests:
 	  KUBEVIRT_INSTALLED_NAMESPACE=${KUBEVIRT_INSTALLED_NAMESPACE} ./hack/build-manifests.sh"
 
 cluster-up:
-	./cluster-up/up.sh
+	./hack/cluster-up.sh
```

Comment:
@dhiller Can you please elaborate what is the benefit of creating `hack/cluster-up.sh` on behalf of running `DEPLOY_KUBEVIRT_CDI="true" ./cluster-up/up.sh` from the Makefile?

Reply:
I was thinking that if the need for further additions of default values arise (as ./hack/common.sh itself can not be sourced by cluster-up), we can do this in the new script. And I think this will happen, i.e. with the discussed defaulting of KUBEVIRT_STORAGE.

Reply:
I think that we clearly see a shift from using manifests from kubevirt/kubevirt to using kubevirtci bundles.